README.md

```diff
@@ -7,22 +7,16 @@ nstack is a stack-like merkle datastructure for storing and accessing indexed va
 
 Operations supported are add and remove at the end of the structure, and mutable access to indexed leaves.
 
-# Usage example
-
+## Usage example
 ```rust
+use nstack::NStack;
 
-use kelvin::{annotations::Cardinality, Blake2b};
-
-let n: usize = 256;
-let mut nt = NStack::<_, Cardinality<u64>, Blake2b>::new();
-
-for i in 0..n {
-    nt.push(i).unwrap();
-}
-
-// get a mutable reference to the 128'th element
+let mut nt = NStack::<i32, ()>::new();
 
-let element = &mut *nt.get_mut(128).unwrap().unwrap();
+nt.push(0);
+nt.push(1);
```

Comment:
nitpick - but why is thus usage example less comprehensive than before?

Reply:
I wanted to avoid including `microkelvin` in the usage example. Now that I look at it, it wouldn't add much complexity anyway.